src/components/can_cooperation/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-#set(target CANCooperation)
+set(target "CANCooperation")
```

Comment:
@LuxoftAKutsan, regarding `CANCooperation` module. There exists some commands from it, which has been disabled not in case of marge, but before that. Reason of it is unknown.